chat/anthropic_chat.py

```diff
@@ -0,0 +1,44 @@
+from anthropic import Anthropic
+from .engine import ChatEngine
+from chat.exceptions import ChatResponseGenerationError
+
+
+class ChatAnthropic(ChatEngine):
```

Comment:
This entire code seems to be quite similar to the `openai_chat.py` perhaps we can make it more modular such that we don't need to implement an entirely new class?

what do you think?

Reply:
Hi @arkanalexei, you're right. Let's refactor it so that `openai_chat.py` and `anthropic_chat.py` doesn't contain as much duplicated code.